File: Data Structures & Algorithms/reverse-string/submission-1.cpp
class Solution {
public:
    void reverseString(vector<char>& s) {
        int l = 0, r = s.size()-1;
        int n = s.size();
        while (l < r) {
            char c = s[l];
            s[l] = s[r];
            s[r] = c;
            l++; r--;
        }
    }
    
};

File: Data Structures & Algorithms/string-encode-and-decode/submission-1.py
class Solution:

    def encode(self, strs: List[str]) -> str:
        result = "";
        final = ""
        for i in strs:
            result += i + ','
        for c in range(len(result)-1):
            final += result[c]

        return final

    def decode(self, s: str) -> List[str]:
        output = s.split(',')
        return output
